db/dummy/packet.json

```diff
@@ -0,0 +1,1000 @@
+{"id":1,"name":"Filbert","display_name":"Filbert Bundock","parameters":[{},{}],"published":false}
```

Comment:
`parameters` will be a dictionary rather than an array, so like `"parameters": {a: 1, b: 2}`